docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: Getting Started with Access Monitoring
+description: Learn how to use Access Monitoring.
+---
+
+
+
+Teleport Access Monitoring allows to query and analyze the audit events generated by Teleport.
+It provides a SQL interface to query the audit logs and generate reports.
+Access Monitoring is a powerful tool to understand and analyze the access patterns in your Teleport cluster.
+This guide will help you understand how to use Access Monitoring to monitor access to resources in your Teleport cluster.
+
+
+## Prerequisites
+- Teleport Cloud cluster v14.2.2 or later.
```

Comment:
If this guide is only for cloud, I wonder if we even need to mention the specific version. Maybe we just say "running Teleport 14 or later" since we know we aren't running any v14 tenants on versions this old.